libraries/rush-lib/assets/rush-init/common/config/rush/experiments.json

```diff
@@ -40,5 +40,11 @@
    * If true, the phased commands feature is enabled. To use this feature, create a "phased" command
    * in common/config/rush/command-line.json.
    */
-  /*[LINE "HYPOTHETICAL"]*/ "phasedCommands": true
-}
+  /*[LINE "HYPOTHETICAL"]*/ "phasedCommands": true,
+
+  /**
+   * If true, rush install or rush update implicitly specify --ignore-scripts during pnpm install,
+   * and run install lifecycle scripts by pnpm rebuild --pending after pnpm install successfully.
+   */
+  /*[LINE "HYPOTHETICAL"]*/ "deferredInstallationScripts": true
```

Comment:
Will this eventually be our recommended approach?

Reply:
Yes, I think so